k8s/aat/cluster-00/pcq/pcq-loader.yaml

```diff
@@ -0,0 +1,26 @@
+---
+apiVersion: helm.fluxcd.io/v1
+kind: HelmRelease
+metadata:
+  name: pcq-loader
+  namespace: pcq
+  annotations:
+    flux.weave.works/automated: "true"
+    flux.weave.works/tag.job: glob:prod-*
+spec:
+  releaseName: pcq-loader
+  rollback:
+    enable: true
+  chart:
```

Comment:
Charts must be used from https://github.com/hmcts/hmcts-charts as announced recently

Reply:
Done